src/clients/affiliate.ts

```diff
@@ -47,10 +42,12 @@ export class Affiliate extends JanusClient {
       }
     )
   }
-}
-interface AffiliateInput {
-  name: string
-  id: string
-  salesChannelId: string
-  searchEndpoint: string
+
+  private get routes() {
```

Comment:
I'd suggest keeping this above... I like to see all the routes that a clients connects to when I open its file. What do you think?